app/institutions/dashboard/-components/institutional-users-list/styles.scss

```diff
@@ -306,3 +306,22 @@ input:checked + .slider::before {
     justify-content: flex-end;
     margin-right: 15px;
 }
+
+
+.icon-message {
+    opacity: 0;
+    color: $color-text-blue-dark;
+}
+
+.icon-message:hover {
+    opacity: 1;
+    color: $color-text-blue-dark;
+    border: 0 !important;
```

Comment:
`!important` is being used here because button tags, always override this value in classes (I think) due accessibility reasons. Consider changing this to something else.